src/schema/objects/entities.yaml

```diff
@@ -342,12 +342,12 @@ split:
   display_name: Split
   description: |
     In the case of long data recordings that exceed a file size of 2Gb,
-    `.fif` files are conventionally split into multiple parts.
-    Each of these files has an internal pointer to the next file.
+    `.fif`, `.nwb`, `.nix` files are conventionally split into multiple parts.
```

Comment:
NWB files that exceed 2GB are not "conventionally" split into multiple parts. I don't know if this is true for NIX but I doubt it

Reply:
we should look into https://bids-specification.readthedocs.io/en/stable/appendices/entities.html#split may be it would apply